pkg/tanka/export.go

```diff
@@ -37,8 +38,12 @@ type ExportEnvOpts struct {
 	Merge bool
 	// optional: only export specified Kubernetes manifests
 	Targets []string
-	// optional: options for parsing Environments
-	ParallelOpts ParallelOpts
+	// optional: options to parse Jsonnet
+	JsonnetOpts JsonnetOpts
```

Comment:
All other Opts types embed `tanka.Opts`, which covers `JsonnetOpts`. For consistency, we should do that here as well